Nodejs/Product/Nodejs/Workspace/JsFileDebugLaunchTargerProvider.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Web;
+using Microsoft.NodejsTools.Debugger.DebugEngine;
+using Microsoft.NodejsTools.Project;
+using Microsoft.NodejsTools.Telemetry;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Workspace;
+using Microsoft.VisualStudio.Workspace.Debug;
+using Microsoft.VisualStudio.Workspace.Extensions.VS.Debug;
+using Microsoft.VisualStudio.Workspace.VSIntegration.Contracts;
+
+namespace Microsoft.NodejsTools.Workspace
+{
+    [ExportVsDebugLaunchTarget(ProviderType, new[] { ".js" }, ProviderPriority.Lowest)]
+    internal sealed class JsFileDebugLaunchTargerProvider : LaunchDebugTargetProvider, IVsDebugLaunchTargetProvider, ILaunchDebugTargetProvider
+    {
+        private const string ProviderType = "{0C231F54-C3EF-4719-9CA5-102B1B63A8DC}";
+
+        [ImportingConstructor]
+        public JsFileDebugLaunchTargerProvider(SVsServiceProvider serviceProvider, IVsFolderWorkspaceService workspaceService)
+            : base(serviceProvider, workspaceService)
+        {
+        }
+
+        void IVsDebugLaunchTargetProvider.SetupDebugTargetInfo(ref VsDebugTargetInfo vsDebugTargetInfo, DebugLaunchActionContext debugLaunchContext)
+        {
+            var nodeExe = CheckNodeInstalledAndWarn(debugLaunchContext);
+
+            var nodeVersion = Nodejs.GetNodeVersion(nodeExe);
+            if (nodeVersion >= new Version(8, 0) || NodejsProjectLauncher.CheckDebugProtocolOption())
```

Comment:
>CheckDebugProtocolOption [](start = 74, length = 24)

This is a confusing name.  It looks like `true` means "use the chrome debugger protocol"?

Reply:
I'll check with the diagnostics team if we still need this